presidio-analyzer/Pipfile

```diff
@@ -22,3 +24,4 @@ flake8= {version = ">=3.7.9"}
 pep8-naming = "*"
 flake8-docstrings = "*"
 pre_commit = "*"
+python-dotenv = "*"
```

Comment:
I don't see a .env file?

Reply:
possibly the one in the root folder?

Reply:
The .env is only used for testing locally. Currently this test doesn't run, but we can consider adding env variables to be used during testing if we want. WDYT?

Reply:
You can set but I would mock the TA client

Reply:
Done